client/index.html

```diff
@@ -13,11 +11,11 @@
     <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.4.8/angular-route.js"></script>
 
     <script src="auth/auth.js"></script>
-    <script src="./services/services.js"></script>
-    <script src="app.js"></script>
     <script src="dashboard/search.js"></script>
     <script src="profile/profile.js"></script>
     <script src="landing/landing.js"></script>
```

Comment:
please remove this from your commit as well: angular is not being loaded if this remains